docs/versioning.md

```diff
@@ -0,0 +1,16 @@
+# Versioning
+
+Out of the box, the Pact Broker allows uploading of pact files with semver
+style versions (eg 2.0.1). For our usage, we wanted to be able to upload pact
+files from various branches in addition the released versions so that our
+branch builds of consumers can verify their pactfiles with the producers.
+
+Pact Broker allows us to [implement our own versioning
+scheme](https://github.com/bethesque/pact_broker/wiki/Configuration#version-parser)
+by providing a custom version parser.  We've [used
```

Comment:
```suggestion
by providing a custom version parser. We've [used
```

Reply:
Fixed in https://github.com/alphagov/govuk-pact-broker/pull/123/commits/54d61fbf7ff94d7d824e6ce31f4d6af4fc73cdfc.